scripts/irods/test/test_izonereport.py

```diff
@@ -10,8 +10,13 @@
 from . import session
 from ..test.command import assert_command
 from ..configuration import IrodsConfig
+from . import settings
```

Comment:
It appears that importing settings is not necessary and the tests still run properly.

Reply:
I'm a little confused. Why was `settings` needed? It appears to be used only as a member of `test` (e.g. `test.settings`).

If it's not needed, please remove it.

Reply:
I had copied decorator for skipping certain tests and some of the imports. Specifically from test_ichksum.py which also appears to have an unnecessary import of settings. Maybe that is something to look into in the future?

Reply:
Correct.